design/ceph/external-cluster-monitoring.md

```diff
@@ -0,0 +1,63 @@
+---
+title: external-cluster-monitoring
+target-version: release-1.4
+---
+
+# External cluster monitoring
```

Comment:
Instead of adding this new design doc, what about adding a new section to ceph-external-cluster.md? This feels like an addition to the external cluster design, rather than requiring a new design doc.

Reply:
Sounds good.